app/gui2/src/util/iconList.json

```diff
@@ -0,0 +1 @@
+["add_column","add_row","array_new2","array_new","arrow_down","calendar","chat_gpt2","chat_gpt_mod2","chat_gpt_mod","chat_gpt_segment_mod","chat_gpt_segment","chat_gpt","clean_dataframe","cloud_from","cloud","command2","command_key2","command_key3","command_key","command","convert","ctrl_key","data_input","data_output","enso_logo","find","ghost","google_color","google","in_out2","in_out","join2","join","lab","local_scope2-1","local_scope2","local_scope3","local_scope","location2","location","map_row","microphone","number_input","option_key","parse-1","parse2","parse3","parse4","parse","predict","preparation","right_side_panel","robot","select_column","select_row","shortcut2","shortcut3","shortcut","sort","table_edit","table_from_columns","table_from_rows","text2","text_input","text","time2","time","transform2","transform3","transform4","transform","unstable2","unstable","marketplace","arrow_left","arrow_right_head_only","arrow_right","graph_editor","workflow_play","docs","eye","no_auto_replay","underscore","sort","sort_ascending","sort_descending","fullscreen","exit_fullscreen","eye","compass","show_all","find","path2","geo_map_distance","geo_map_pin"]
```

Comment:
How is this file generated? Do we need to update it when adding new icons? How to do that?

Reply:
i manually used find + replace to generate this, but i've now added a generation script to generate both this file, and a new file that exports a type containing every icon name as well.
i've also added a new command `npm run generate` to make it easy to update both files when adding new icons